tree/dataframe/inc/ROOT/RDF/RDisplay.hxx

```diff
@@ -90,17 +91,32 @@ private:
    size_t fNMaxCollectionElements = 10; // threshold on number of elements in collections to be Print()
 
    ////////////////////////////////////////////////////////////////////////////
-   /// Appends a cling::printValue call to the stringstream.
+   /// Appends a string if the T type is an arithmetic type.
    /// \tparam T the type of the event to convert
    /// \param[in] stream Where the conversion function call will be chained.
    /// \param[in] element The event to convert to its string representation
    /// \param[in] index To which column the event belongs to
    /// \return false, the event is not a collection
-   template <typename T, std::enable_if_t<!ROOT::Internal::RDF::IsDataContainer<T>::value, int> = 0>
+   template <typename T, typename std::enable_if_t<std::is_floating_point<T>::value && !ROOT::Internal::RDF::IsDataContainer<T>::value, int> = 0>
```

Comment:
I'm restricting it to floating point type to make sure the booleans are treated as they are now.